### PR TITLE
Update d7 v17

### PR DESCRIPTION
**Pull request by congoblue**
Changes to add D7 into the manual

### CODE REVIEW

**Review by owaits — CHANGES_REQUESTED** [review]

File: docs/about-the-consoles/fader-wings.md [hunk 1]
```diff
@@ -44,10 +44,10 @@ position, allowing you to attach two of the wings to a console. The switch posit
 is only read on power-up.
 
 The Tiger Touch Wing is designed to fit next to the Tiger Touch console
-but can also be used with the D9, the Sapphire Touch, the Arena and the
+but can also be used with the D9, D7, the Sapphire Touch, the Arena and the
```
Comment (PeterBuddAvo) [inline docs/about-the-consoles/fader-wings.md:47]: The Tiger Touch Wing will not be supported by the D7.

File: docs/quick-start/dmx-network-setup.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ one of 64 **DMX Lines** which may be routed to the XLR sockets on the console or
 over Ethernet (see [DMX Output Mapping](../system-settings/dmx-output-mapping.md) for details).
 
 - If using the T1 / T2 / T3 hardware the output is limited to the first one, two or sixteen universes respectively.
-A console would normally output up to 16 universes itself, further Titan Network Processors (TNP) units would be connected to the network using TitanNet to control up to 64 universes. For simple programming or rigs with lots of space in the patch you may be able to output more than 16 universes from a console but operating speed may be affected.
+A console would normally output up to 16 universes itself (32 for D7), further Titan Network Processors (TNP) units would be connected to the network using TitanNet to control up to 64 universes. For simple programming or rigs with lots of space in the patch you may be able to output more than 16 universes (32 for D7) from a console but operating speed may be affected.
```
Comment (PeterBuddAvo) [inline docs/quick-start/dmx-network-setup.md:15]: The D9 is also specified as running 32 universes without additional processors.

File: docs/system-settings/dmx-output-mapping.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ over Ethernet.
 All consoles can control up to 64 universes except **T1** which is limited to
 one single universe (fixed at universe 1), **T2** to two universes (fixed at universes 1-2) and **T3** to sixteen universes (fixed at universes 1-16 - expandable with optional extra licence).
 
-The console itself will generate DMX output for up to 16 universes, above that you can network the console to one
+The console itself will generate DMX output for up to 16 universes (32 for D7), above that you can network the console to one
```
Comment (PeterBuddAvo) [inline docs/system-settings/dmx-output-mapping.md:16]: D7 and D9 both support 32 universes out of the box.

File: docs/system-settings/external-displays.md [hunk 4]
```diff
@@ -8,8 +8,8 @@ import Keys from '@site/src/components/key.ts';
 import Video from '@site/src/components/video.tsx';
 
 For extra workspace you can connect external monitors to the monitor ports
-provided on the console (HDMI on Diamond 9, DVI on other consoles, except Pearl Expert and Tiger Touch 1
-which are VGA). The Diamond 9 and Sapphire
+provided on the console (HDMI on Diamond 9 and Diamond 7, DVI on other consoles, except Pearl Expert and Tiger Touch 1
```
Comment (PeterBuddAvo) [inline docs/system-settings/external-displays.md:11]: The ports on the back of a D7 are display port.

File: docs/titan-basics/other-parts-of-the-touch-screen.md [hunk 5]
```diff
@@ -75,7 +75,7 @@ meet in the middle.
 
 ![Sapphire Tools Menu](/docs/images/Sapphire-Tools-Menu.png)
 
-On Diamond 9 the toolbar is replaced by a small <Keys.ContextKey>Tools</Keys.ContextKey> button at the top
+On Diamond 9 / Diamond 7 the toolbar is replaced by a small <Keys.ContextKey>Tools</Keys.ContextKey> button at the top
```
Comment (PeterBuddAvo) [inline docs/titan-basics/other-parts-of-the-touch-screen.md:78]: On the D7 it is on the top left of the left hand screen on a 2 screen or middle screen on a 3 screen.

File: docs/titan-basics/workspace-windows.md [hunk 6]
```diff
@@ -158,7 +158,7 @@ You can save workspace layouts, allowing you to reconfigure either all the windo
 screen, or all the windows on all the screens, at the touch of a button.
 
 Workspace layouts are saved to the Workspaces touch buttons to the left
-of the A-G menu buttons and above the wheel displays (except on Diamond 9, Sapphire Touch and Titan Go where
+of the A-G menu buttons and above the wheel displays (except on Diamond 9, Diamond 7,  Sapphire Touch and Titan Go where
```
Comment (PeterBuddAvo) [inline docs/titan-basics/workspace-windows.md:161]: for A-G buttons do you mean the softkey buttons? I think these are generally referred to as soft key buttons elsewhere.

File: docs/titan-reference/button-reference.md [hunk 7]
```diff
@@ -41,12 +41,12 @@ For the <Keys.HardKey>@</Keys.HardKey> buttons next to the wheels see [Wheel @](
...
 
 ## <Keys.HardKey>Align</Keys.HardKey> 
- (D9 and T3 only) Allows copying of attributes between fixtures. On other consoles this function is accessed using <Keys.HardKey>ML Menu</Keys.HardKey>. See [Align Fixtures](../controlling-fixtures/changing-fixture-attributes.md#align-fixtures).
+ (D9/D7 and T3 only) Allows copying of attributes between fixtures. On other consoles this function is accessed using <Keys.HardKey>ML Menu</Keys.HardKey>. See [Align Fixtures](../controlling-fixtures/changing-fixture-attributes.md#align-fixtures).
 
 
 
 ## <Keys.HardKey>All</Keys.HardKey> 
- Used to start and end pattern selection (such as odd/even) within a range of selected fixtures. See [Selecting fixtures using a pattern](../controlling-fixtures.md#selecting-fixtures-using-a-pattern). (On D9 labelled <Keys.HardKey>Pattern/All</Keys.HardKey>)
+ Used to start and end pattern selection (such as odd/even) within a range of selected fixtures. See [Selecting fixtures using a pattern](../controlling-fixtures.md#selecting-fixtures-using-a-pattern). (On D9/D7 labelled <Keys.HardKey>Pattern/All</Keys.HardKey>)
```
Comment (PeterBuddAvo) [inline docs/titan-reference/button-reference.md:49]: Also Pattern/All on the T3.

File: docs/titan-basics/workspace-windows.md [hunk 8]
```diff
@@ -117,7 +117,7 @@ On all consoles apart from Sapphire Touch and Titan Go, **context menu
 buttons** appear which contain useful functions for the current
 active window and change depending on the window. 
 
-On Diamond 9 the context menu buttons are in the top left hand corner of the right
+On Diamond 9 and Diamond 7 the context menu buttons are in the top left hand corner of the right
```
Comment (PeterBuddAvo) [inline docs/titan-basics/workspace-windows.md:120]: The context menu on the D7 is not in the same place as on the D9. It is just in each individual window more similar to the sapphire.

File: docs/titan-basics/workspace-windows.md [hunk 9]
```diff
@@ -275,7 +275,7 @@ button at a glance. Select the <Keys.SoftKey>Halo</Keys.SoftKey> option on the S
 
 If you set a [halo on fixture buttons](../patching/changing-the-patch.md#fixture-button-halo), the halo colour is also used in other windows; fixture buttons can also have an automatic halo to help identify different types of fixture, which is also used in the patch window.
 
-On the Diamond 9, the halo colour set on playbacks is used in the level bargraph lighting and the playback indicator.
+On the Diamond 9 and Diamond 7, the halo colour set on playbacks is used in the level bargraph lighting and the playback indicator.
```
Comment (PeterBuddAvo) [inline docs/titan-basics/workspace-windows.md:278]: I think bar graph is normally 2 words?

As previously mentioned that isn't what is on the D7 but this sentence could be interpreted either way.

**Review by PeterBuddAvo — COMMENTED** [review]
Sorry bunch of changes that I didn't spot last time I went through this. Hopefully these are the last ones.

**Pull request merged**